### PR TITLE
say byte to hash routing

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: spot-client/src/common/utils/window-handler.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default {
      * @returns {string}
      */
     getBaseUrl() {
```
Comment (paweldomas) [inline spot-client/src/common/utils/window-handler.js]: I feel like this change violates the definition, but not sure

Comment (virtuacoplenny) [reply spot-client/src/common/utils/window-handler.js]: seems okay. window handler only exists to hide window.location related accessing for the unit tests that don't exist.

File: spot-client/src/spot-tv/ui/views/home.js [hunk 2]
```diff
@@ -189,7 +189,7 @@ export class Home extends React.Component {
      */
     _onOpenRemote() {
         const baseUrl = windowHandler.getBaseUrl();
-        const url = `${baseUrl}#/?code=${this.props.joinCode.toUpperCase()}`;
+        const url = `${baseUrl}/${this.props.joinCode.toUpperCase()}`;
```
Comment (virtuacoplenny) [inline spot-client/src/spot-tv/ui/views/home.js]: We should hide this one day in some kind of routing helper. Did you happen to find a decent place to shove it?

File: spot-client/src/spot-remote/ui/views/join-code-entry.js [hunk 3]
```diff
@@ -65,15 +65,16 @@ export class JoinCodeEntry extends React.Component {
         this.props.dispatch(setJoinCode(''));
         remoteControlService.disconnect();
 
-        const queryParams = new URLSearchParams(this.props.location.search);
-        const code = queryParams.get('code');
+        const { pathname } = this.props.location;
+        const codeMatch = pathname.match(new RegExp('^/(\\w{6})$'));
+        const code = codeMatch && codeMatch[1];
```
Comment (virtuacoplenny) [inline spot-client/src/spot-remote/ui/views/join-code-entry.js]: So the default already goes down to the remote control page?

Comment (paweldomas) [reply spot-client/src/spot-remote/ui/views/join-code-entry.js]: yes, here: https://github.com/jitsi/jitsi-meet-spot/blob/master/spot-client/src/app.js#L158

It doesn't have a path defined, that's why

File: spot-client/src/spot-tv/ui/views/home.js (hunk 2 above)
Comment (paweldomas) [reply spot-client/src/spot-tv/ui/views/home.js]: No, but routing helper sounds good to me. Do you want me to create one now ?

Comment (virtuacoplenny) [reply spot-client/src/spot-tv/ui/views/home.js]: Nah, not for this PR.

**Pull request merged**